docs/SDL/OnSDLConsentNeeded/index.md

```diff
@@ -1,15 +1,48 @@
 ## OnSDLConsentNeeded
+Type
+: Notification
 
+Sender
+: SDL
 
-### Notification
+Purpose
+: Send from SDL to HMI to notify that data consent is needed for device
+
+The notification is applicable only for the case when the device is consented by the user and the user requests Policy Table update from HMI.
+
+!!!
+MUST:   
+1) Initiate the sequence of device data consent on getting OnSDLConsentNeeded.   
+2) Upon getting _OnSDLConsentNeeded_, request _SDL.GetUserFriendlyMessages_ from SDL.   
+3) Display the 'device consent' dialog after getting _SDL.GetUserFriendlyMessages_response_ from SDL.   
+4) Send _OnAllowSDLFunctionality_ (true or false) after and depending on User's choice on HMI.   
+
+For more details see the following diagrams:     
+   - [The User does NOT consent the device](../activateapp/assets/user%20does%20not%20consent%20the%20device1.png).   
+   - [The User consents the device](../activateapp/assets/user%20consents%20the%20device2.png).
```

Comment:
These diagrams need to be outside of the `!!! MUST` block, and they should be in the format of

```
|||
The User does NOT consent the device
[OnSDLConsentNeeded](../activateapp/assets/user%20does%20not%20consent%20the%20device1.png)
|||
```